docs/game_sdk/Networking.md

```diff
@@ -285,20 +285,20 @@ var networkManager = discord.GetNetworkManager();
 var lobbyManager = discord.GetLobbyManager();
 var userManager = discord.GetUserManager();
 
-var me;
+Discord.User currentUser;
 var otherUserPeerId;
 var lobbyId;
 
 // Get yourself
-me = userManager.GetCurrentUser();
+currentUser = userManager.GetCurrentUser();
 
 // This will fire once you connect to the lobby
 // Telling you which route is yours
 networkManager.OnRouteUpdate += route =>
 {
-  var txn = lobbyManager.GetMemberUpdateTransaction();
-  txn.SetMetadata("route", route);
-  lobbyManager.UpdateMember(lobbyId, me.Id, txn, (result =>
+  var txn = lobbyManager.GetMemberUpdateTransaction(lobbyId, currentUser.Id);
+  txn.SetMetadata("metadata.route", route);
```

Comment:
You don't need to explicitly have `metadata.` when setting metadata, only when `GET`ing it.

Reply:
From my experience using the metadata, I did previously set it as just ` txn.SetMetadata("route", route);` or for the peer_Id as well, but it would result from discord saying 'Not Found'. So i set it as that to completely make sure the wording is the same, and in my case/experience Discord returns the peer_id and route properly... So hence why I have it as such.

Reply:
I’d like to test this myself before changing the docs just to be sure if that’s alright. Thanks for your patience! I’ll test this tomorrow.

Reply:
Anything new regarding this?

Reply:
@Amorelandra was testing for a repro. Any luck?

Reply:
When accessing `metadata` methods you should not need a `metadata.` prefix. Metadata is just a key-value store, so if you set `metadata.route` then you would have to retrieve with `metadata.route` in `GetMemberMetadataValue`.

The `metadata.` prefix comes into play when you are doing filtering/sorting on lobbies, since `metadata.` triggers a comparison against the metadata on a lobby

Reply:
> 
> 
> When accessing `metadata` methods you should not need a `metadata.` prefix. Metadata is just a key-value store, so if you set `metadata.route` then you would have to retrieve with `metadata.route` in `GetMemberMetadataValue`.
> 
> The `metadata.` prefix comes into play when you are doing filtering/sorting on lobbies, since `metadata.` triggers a comparison against the metadata on a lobby

I see Night! That DOES make more sense how the prefix comes into play, but what i still don't get is why without that prefix being set to the data of the user, it gives 'result: not found' this is the workaround I've come across... So not sure...? I'd be glad to reverse this if that is the case.

Reply:
I can confirm that what night said is true. Here is the example code for a Unity sample app:

```cs
discord = new Discord.Discord(APP_ID, (System.UInt64)Discord.CreateFlags.Default);
    discord.SetLogHook(Discord.LogLevel.Debug, (Discord.LogLevel level, string message) =>
    {
      Debug.Log(message);
    });

    lobbyManager = discord.GetLobbyManager();
    var txn = lobbyManager.GetLobbyCreateTransaction();
    txn.SetMetadata("mmr", "1000");

    lobbyManager.CreateLobby(txn, (Discord.Result res, ref Discord.Lobby lobby) =>
    {
      Debug.Log(lobby.Id);
      Debug.Log("Metadata: " + lobbyManager.GetLobbyMetadataValue(lobby.Id, "mmr"));
    });
```

![image](https://user-images.githubusercontent.com/4193854/72394861-8765d100-36ec-11ea-980a-61b03f335425.png)

so yea please update accordingly :)

Reply:
Reversing!